rabbitmq-ballerina/client.bal

```diff
@@ -197,7 +197,7 @@ public isolated client class Client {
     #               (#https://www.rabbitmq.com/resources/specs/amqp0-9-1.pdf))
     # + closeMessage - A message indicating the reason for closing the channel
     # + return - A `rabbitmq:Error` if an I/O error occurred or else `()`
-    isolated function close(int? closeCode = (), string? closeMessage = ()) returns Error? =
+    isolated remote function close(int? closeCode = (), string? closeMessage = ()) returns Error? =
```

Comment:
Shall we fix the API usage in the docs section as well? Line #193 and #208

Reply:
Fixed in 9532b4871953571443db51f55210d87b2e14381d